README.md

```diff
@@ -259,17 +259,23 @@ The `SwiftGen.playground` available in this repository will allow you to play wi
 
 This allows you to have a quick look at how typical code generated by SwiftGen looks like, and how you will then use the generated constants in your code.
 
-### Markdown files
+### Dedicated Documentation in Markdown
 
-There are also a lot of documentation in the form of Markdown files in this repository and the related [StencilSwiftKit](https://github.com/SwiftGen/StencilSwiftKit) repo as well. Be sure to check the "Documentation" folder of each repository.
+There are also a lot of documentation in the form of Markdown files in this repository and the related [StencilSwiftKit](https://github.com/SwiftGen/StencilSwiftKit) repo as well.
 
-### Wiki
+Be sure to [check the "Documentation" folder](Documentation/) of each repository.
 
-You can also see in the [wiki](https://github.com/SwiftGen/SwiftGen/wiki) some additional documentation, about:
+Especially, in addition to the previously mentionned [Migration Guide](./Documentation/MigrationGuide.md) and [Configuration File](./Documentation/ConfigFile.md) documentation, the `Documentation/` folder in the SwiftGen repository also includes documentation on how to:
 
-* how to [integrate SwiftGen in your Continuous Integration](https://github.com/SwiftGen/SwiftGen/wiki/Continuous-Integration) (Travis-CI, CircleCI, Jenkins, …)
-* how to [integrate in your Xcode project](https://github.com/SwiftGen/SwiftGen/wiki/Integrate-SwiftGen-in-an-xcodeproj) so it rebuild the constants every time you build
-* …and more.
+* [Integrate SwiftGen in your Xcode project](./Documentation/Articles/Xcode-Integration.md) — so it rebuilds the constants every time you build
+* [Configure SwiftLint to help your developers use constants generated by SwiftGen](./Documentation/Articles/SwiftLint-Integration.md)
+* [Create a custom template](Creating-your-templates.md), and [watch a folder to auto-regenerate an output every time you save the template you're working on](./Documentation/Articles/Watch-a-folder-for-changes.md)
+* …and more
+
+That folder also contains:
```

Comment:
I'd move these two to the top, they're the most important ones. The articles can come after?